docs/admin/system-information.rst

```diff
@@ -834,9 +834,16 @@ Recovery expected data nodes
 
 The check for the
 :ref:`gateway.expected_data_nodes <gateway.expected_data_nodes>` setting checks
-that the number of data nodes that should be waited for the immediate
-cluster state :ref:`recovery <gloss-shard-recovery>`. This value must be equal
-to the maximum number of data nodes in the cluster.
+if the setting match the actual number of data nodes present in the cluster.
+The value of this settings is checked against 
+:ref:`gateway.recover_after_data_nodes
+<gateway.recover_after_data_nodes>` (or 
+:ref:`gateway.recover_after_nodes <gateway.recover_after_nodes>`
+) during startup to decide if the cluster can proceed with 
+:ref:`recovery <gloss-shard-recovery>`.
+If the actual number of nodes is below the number expected this is flagged to
+indicate some node[s] are down. If the actual number is greater this is 
```

Comment:
I think you can drop the "[s]", otherwise you would also need to use "is/are" later in the sentence.
```suggestion
indicate some nodes are down. If the actual number is greater, this is 
```